admin/config/locales/en.yml

```diff
@@ -286,6 +286,7 @@ en:
             views: views
         edit:
           client_id_note: Identifies the category for product import
+          default_sort_note: 'This category has featured products. When a category has featured products, this sort will appear as the "Featured" option on the storefront.'
```

Comment:
```suggestion
          default_sort_note: 'Since this category has featured products, this sort controls the products listed after the selected featured products.'
```